debian/control

```diff
@@ -8,21 +8,16 @@ Build-Depends: debhelper (>= 10),
                python3,
                python3-setuptools,
                python3-gi,
-               libgtk-3-dev,
-               exiftools
-X-Python3-Version: >= 3.6.9
+               libgtk-3-dev
+X-Python3-Version: >= 3.6
 Standards-Version: 3.9.3
 
 Package: com.github.hezral.inspektor
 Architecture: any
 Depends: ${python3:Depends},
-         dh-python,
-         python3-dev,
+         ${misc:Depends},
          python3,
-         python3-setuptools,
          python3-gi,
-         libgtk-3-dev,
-         exiftools,
-         ${misc:Depends}
+         exiftools
```

Comment:
This package is not available in bionic/Hera, but `exiftool` is provided by `libimage-exiftool-perl`.

Reply:
Thanks, you're right, overlooked the actual package name. 
I assumed travisci would have picked it up when i saw there weren't any build errors.